WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsDetails.swift

```diff
@@ -1,19 +1,28 @@
 import SwiftUI
 
-struct OrderCustomFieldsDetails: View {
+struct CustomFieldsDetails: View {
     @Environment(\.presentationMode) var presentationMode
 
-    let customFields: [OrderCustomFieldsViewModel]
+    let viewEditCustomFieldsInProductsAndOrdersEnabled: Bool
```

Comment:
Super nit: the naming is a bit verbose making it hard to follow. How about `isEditable`?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13639/commits/e23c6d26008ae66a7f7dd5818d604828245b62de